torch/csrc/jit/interpreter.h

```diff
@@ -43,13 +43,10 @@ struct TORCH_API Code {
 
 struct InterpreterState {
   InterpreterState(const Code & code);
-  // advance the interpreter state by running one stage. Returning the
-  // outputs for that stage, suspending the computation.
-  // Call this function again continues computation where it left off.
-  void runOneStage(Stack & stack);
+  void run(Stack & stack);
   ~InterpreterState();
   // create a copy of InterpreterState with its current state
-  // used when retain_graph=True so that stages can be re-run
+  // used when retain_graph=True
```

Comment:
This comment is outdated. This was needed to have a non-destructive way to run stage 1 (or later), but that's no longer necessary. I think it's safe to remove the cloning ability from `InterpreterState`